### ISSUE TITLE
Fullscreening video playback may freeze Librewolf

### ISSUE THREAD

**Issue by iacore** [QubesOS/qubes-issues#7603]
tl;dr, video playback is buggy and I can't figure out why

### Qubes OS release
r4.1 with modification

### Brief summary

Maximizing Librewolf while playing videos may froze the window.

- other windows are still responsive
- cannot close the froze window. Seems like gui-agent responsible for the window is broken
- can close the app with `pkill` from terminal

### Steps to reproduce
Don't know how to reproduce.
Youtube works fine, but some other video websites with bad JS video player seem to cause this.

I tried to debug this, but logging is totally insufficient (no useful info in logs). This probably has something to do with the LLVM OpenGL software rasterizer.

### Expected behavior
Websites should not brick the browser window.

### Actual behavior
On some websites, when playing video in librewolf, maximizing window will froze the window (from user's point of view). The process in AppVM still runs fine, and has normal audio playback. Only mapped window buffer is frozen, and it cannot process input events from dom0. Weird thing is, the window is not closed immediately, which will happen if the gui-agent responsible for mapping this window in AppVM quits.

**Comment by locriacyber** [comment]
There is another (maybe related) issue. When developing games with raylib and use `ls *.nim | entr -rcc nim r main.nim` to rerun game on file save, after 20x times of reload, the window becomes pure white (it should at least draw something). This should be related to OpenGL context being broken, since closing some graphical apps and rerun the game seem to get functioning OpenGL context again.
